File: Programar em Java/15-Switch.java
// YouTube: https://youtu.be/445d_X4N6OI
// Publicação: https://caffeinealgorithm.com/blog/20220224/switch-em-java/

package com.caffeinealgorithm.programaremjava;

public class Switch {
  public void Run() {
    char caso = 'D';

    switch (caso) {
      case 'A':
        System.out.println("O caso A existe.");
        break;
      case 'B':
        System.out.println("O caso B existe.");
        break;
      case 'C':
        System.out.println("O caso C existe.");
        break;
      default:
        System.out.printf("O caso %c não existe.", caso);
        break;
    }
  }
}

// O caso D não existe.

File: Programar em C#/15-Switch.cs
// YouTube: https://youtu.be/vqC8km6r22k
// Publicação: https://caffeinealgorithm.com/blog/switch-em-csharp/

using System;

namespace Base {
  class Switch {
    public void Run() {
      char caso = 'D';

      switch (caso) {
        case 'A':
          Console.WriteLine("O caso A existe.");
          break;
        case 'B':
          Console.WriteLine("O caso B existe.");
          break;
        case 'C':
          Console.WriteLine("O caso C existe.");
          break;
        default:
          Console.WriteLine($"O caso {caso} não existe.");
          break;
      }
    }
  }
}

// O caso D não existe.
